event_phone/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module validate phone numbers in the Attendee form of the Event module, just like the
```

Comment:
s /This module validate/This module validates/

Reply:
fixed

Reply:
not too sure what is the problem with Travis https://travis-ci.com/github/OCA/connector-telephony/jobs/505673501#L549 but it's not passing sadly..;